cli/submit.py

```diff
@@ -2,27 +2,42 @@
 import json
 import requests
 
+from cli import LSRI_CLIENT_ID
 from ebi_eva_common_pyutils.logger import logging_config
 from getpass import getpass
+from cli.lsri_auth import LSRIAuth
 
 logger = logging_config.get_logger(__name__)
 
 
 class StudySubmitter:
     def __init__(self, ENA_AUTH_URL="https://www.ebi.ac.uk/ena/submit/webin/auth/token",
-                 SUBMISSION_INITIATE_ENDPOINT ="http://www.ebi.ac.uk/eva/v1/submission/initiate/webin"):
+                 SUBMISSION_INITIATE_WEBIN_URL ="http://www.ebi.ac.uk/eva/v1/submission/initiate/webin",
+                 SUBMISSION_INITIATE_LSRI_URL ="http://www.ebi.ac.uk/eva/v1/submission/initiate/lsri"):
         self.ENA_AUTH_URL = ENA_AUTH_URL
-        self.SUBMISSION_INITIATE_ENDPOINT = SUBMISSION_INITIATE_ENDPOINT
+        self.SUBMISSION_INITIATE_WEBIN_URL = SUBMISSION_INITIATE_WEBIN_URL
+        self.SUBMISSION_INITIATE_LSRI_URL = SUBMISSION_INITIATE_LSRI_URL
```

Comment:
I think these default constant should be set ad the module level and parameter names should be lover cased.

Something like 
```python
ENA_AUTH_URL="https://www.ebi.ac.uk/ena/submit/webin/auth/token"
SUBMISSION_INITIATE_WEBIN_URL ="http://www.ebi.ac.uk/eva/v1/submission/initiate/webin"
SUBMISSION_INITIATE_LSRI_URL ="http://www.ebi.ac.uk/eva/v1/submission/initiate/lsri"


class StudySubmitter:
    def __init__(self, ena_auth_url=ENA_AUTH_URL,
                 submission_initiate_webin_url=SUBMISSION_INITIATE_WEBIN_URL 
                 submission_initiate_lsri_url=SUBMISSION_INITIATE_LSRI_URL):

```

Reply:
Addressed